packages/pool-metadata-gql/codegen.ts

```diff
@@ -3,16 +3,17 @@ import { CodegenConfig } from "@graphql-codegen/cli";
 const BASE_URL = "https://api.thegraph.com/subgraphs/name/bleu-llc";
 
 export enum Network {
-  // mainnet = "mainnet",
-  // polygon = "polygon",
-  // arbitrum = "arbitrum",
+  mainnet = "mainnet",
+  polygon = "polygon",
+  arbitrum = "arbitrum",
   goerli = "goerli",
 }
 
+// TODO: https://linear.app/bleu-llc/issue/BAL-131/deploy-contracts-in-all-networks-that-balancer-is-deployed
 export const ENDPOINTS = {
-  // [Network.mainnet]: `${BASE_URL}/balancer-v2`,
-  // [Network.polygon]: `${BASE_URL}/balancer-polygon-v2`,
-  // [Network.arbitrum]: `${BASE_URL}/balancer-arbitrum-v2`,
+  [Network.mainnet]: `${BASE_URL}/balancer-pools-metadata`,
+  [Network.polygon]: `${BASE_URL}/balancer-pools-metadata`,
+  [Network.arbitrum]: `${BASE_URL}/balancer-pools-metadata`,
   [Network.goerli]: `${BASE_URL}/balancer-pools-metadata`,
```

Comment:
ok, so we added these endpoints here as an example but we're not fully leveraging them 👍 sgtm. Could we maybe then just add the full list of networks we're going to need to deploy to? It could use the same endpoints etc.